docs/site/Getting-started.md

```diff
@@ -42,13 +42,13 @@ Answer the prompts as follows:
 ? Project root directory: (getting-started)
 ? Application class name: StarterApplication
 ? Select features to enable in the project:
-❯◉ Enable tslint
- ◉ Enable prettier
- ◉ Enable mocha
- ◉ Enable loopbackBuild
- ◉ Enable vscode
- ◉ Enable repositories
- ◉ Enable services
+❯◉ Enable tslint: adds a linter with pre-configured lint rules
+ ◉ Enable prettier: installs prettier for consistent code formatting
+ ◉ Enable mocha: installs mocha as a dependency
+ ◉ Enable loopbackBuild: sets up TypeScript compiler with other features
```

Comment:
I believe we are always setting TypeScript compilation. This flag determines whether we use `@loopback/build` helpers or call `tsc` directly. IIRC, this affects other build tooling like `mocha` (vs. `lb-mocha`) and `tslint` (vs. `lb-tslint`) too.